vector/v.edit/testsuite/select_all_flag.sh

```diff
@@ -15,7 +16,7 @@ v.edit -r map=vedit_test tool=catadd layer=2 cats=10
 
 expected="10
 10"
-out=$(v.category option=print layer=2 input=vedit_test)
+out=$(v.category option=print layer=2 input=vedit_test| tr -d '\r')
```

Comment:
BTW, skipping all .sh-based tests on Windows would be a fair solution I think.

Reply:
I have no strong opinion here. MSYS has been dropped, so it is very unlikely that users will run shell scripts in real world cases. And I am not sure this is a real MS Windows test in that sense.
However, most of the shell tests work. So maybe it does not hurt to have them running for MS Windows as well...

Reply:
Okay, please add comment that the \r part is for Windows, so we know when to remove it.

However, I think the role of Bash/sh/... tests is to provide option for "quick and dirty" tests and for very specific cases, so supporting only Linux or unix is fine. Something to keep in mind.